util/build-gnu.sh

```diff
@@ -164,6 +164,8 @@ sed -i -e "s|rm: cannot remove 'a/1'|rm: cannot remove 'a'|g" tests/rm/rm2.sh
 
 sed -i -e "s|removed directory 'a/'|removed directory 'a'|g" tests/rm/v-slash.sh
 
+sed -i -e "s|---dis ||g" tests/tail-2/overlay-headers.sh
```

Comment:
could you pleas comment the why ?

Reply:
Please have a look at the commit message: https://github.com/uutils/coreutils/pull/3618/commits/70b481c0d5d88b5058bfc97c40fd9ea04eaefd15

Reply:
On second thought, I think we should remove `---dis`/`---disable-inotify` from all the relevant GNU tests. Because:

a) This PR improves the situation immensely, however there are still tests that fluctuate from time to time between PASS/FAIL because of "unexpected timeout" issues related to polling.
b) This feature is rarely used anyway (Polling instead of inotify is just a fallback for when inotify is not available and the flags are only there for test purposes)

What do you think?

Reply:
I trust your judgment. I don't mind removing options that don't matter to us.

Reply:
About my request for comment. I meant in the code itself.
it is much less convenient to look at the commit history

Reply:
I added a comment in the code itself.